### DISCUSSION TITLE
Minimum gpus for pretraining Yi-34

### DISCUSSION THREAD

**Discussion by chuan298** [axolotl-ai-cloud/axolotl#939] [Q&A]
Hi team and @winglian ,
I get OOM error when pretraining Yi-34 with 8*A100 80GB, flash_attn, deepspeed zero3, sequence_len=2048. So what is minimum gpus for this?

**Comment by NanoCode012** [comment]
Hey, can you try enable cpu offload and reduce batch size? Theoretically, it should fit.
